bip-0158.mediawiki

```diff
@@ -264,7 +264,10 @@ against the decompressed GCS contents. See
 === Block Filters ===
 
 This BIP defines one initial filter type:
```

Comment:
Now there are *two* initial filter types.

Reply:
Thank you!